source/features/more-file-links.tsx

```diff
@@ -27,9 +30,34 @@ function handleMenuOpening({delegateTarget: dropdown}: DelegateEvent): void {
 	);
 }
 
+function handleMenuOpeningReact(): void {
+	const viewFile = $('[class^="prc-ActionList-ActionListItem"]:has(.octicon-eye)');
+	const fileLink = $('a', viewFile).href;
+
+	const getMenuItem = (name: string, route: string, icon?: React.JSX.Element): HTMLElement => {
+		const menuItem = viewFile.cloneNode(true);
+		const link = $('a', menuItem);
+		link.href = new GitHubFileURL(fileLink).assign({route}).href;
+		link.dataset.turbo = String(route !== 'raw');
+		$('[class^="prc-ActionList-ItemLabel"]', menuItem).textContent = `View ${name}`;
+		$('[class^="prc-ActionList-LeadingVisual"]', menuItem).replaceChildren(icon ?? '');
```

Comment:
`replaceChildren` accepts `undefined` (I don't know if the types allow that)


```suggestion
		$('[class^="prc-ActionList-LeadingVisual"]', menuItem).replaceChildren(icon);
```